buildpack-deps/Dockerfile.fedora.tpl

```diff
@@ -0,0 +1,26 @@
+FROM #{FROM}
+
+RUN dnf install -y \
+		autoconf \
+		make automake gcc gcc-c++ kernel-devel \
```

Comment:
@petrosagg `@development-tools` will install too many packages that won't be necessary in our base images so I don't want to use it.
I will sort the package name, rebase then merge this PR